pkg/agent/agent.go

```diff
@@ -336,6 +336,8 @@ func (i *Initializer) initOpenFlowPipeline() error {
 		// TODO: introduce a deterministic mechanism through which the different entities
 		// responsible for installing flows can notify the agent that this deletion
 		// operation can take place.
+		// TODO: For deterministic mechanism, restartFullSyncWaitGroup could be part of the solution
```

Comment:
`restartFullSyncWaitGroup` doesn't seem to exist

also:
s/For deterministic mechanism,/As a deterministic mechanism,
s/are synced once/have been synced once

Reply:
Done.

> Some comments. I noticed that you added quite a few `klog.Infof` messages. Do you feel they all belong at the default verbosity level or should be moved to verbosity 2?
> 
> Since there is no issue referenced, do you think you could edit the commit message & PR description to indicate which issue this is solving (or if it's just an optimization).

Will evaluate and update this.